dspace/config/item-submission.xml

```diff
@@ -18,6 +18,12 @@
 	<!-- for handle "default". -->
 	<submission-map>
 		<name-map collection-handle="default" submission-name="traditional" />
+		<name-map collection-handle="123456789/6" submission-name="People"/>
+		<name-map collection-handle="123456789/7" submission-name="Project"/>
+		<name-map collection-handle="123456789/8" submission-name="OrgUnit"/>
+		<name-map collection-handle="123456789/28" submission-name="Journals"/>
+		<name-map collection-handle="123456789/29" submission-name="JournalVolumes"/>
+		<name-map collection-handle="123456789/30" submission-name="JournalIssues"/>
```

Comment:
Lots of test data that we'll have to remove (eventually)

Reply:
The goal for this was to create an easy test setup with some initial items and collections, and to remove it later-on
But upon further review, it seems to also be unable to create that test setup, so I think we can better remove it and identify another solution to make it easier to populate a test repository for verifying the functionality

Reply:
This should be resolved using commit b37d0e216718316ba202f34556f53344cd0346b8